conda/recipes/ptxcompiler/meta.yaml

```diff
@@ -13,16 +13,22 @@ source:
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: cuda_{{ cuda_version }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
+    - {{ compiler("c") }} 9
+    - {{ compiler("cxx") }} 9
+  host:
     - python
     - pip
+    - cudatoolkit {{ cuda_version }}.*
   run:
     - python
     - numba >=0.54
+    - cudatoolkit >={{ cuda_major ~ ".0" }},<={{ cuda_version }}
```

Comment:
This would be something like 

```
- cudatoolkit >=11.0,<=11.6.1
```

With `ptxcompiler`, the latest version we can support is the latest CUDA version built with

Reply:
It is [a concatenation operator on `str`s in Jinja]( https://jinja.palletsprojects.com/en/3.0.x/templates/?highlight=tilde#other-operators )

Reply:
This operator is the string concatenation in Jinja2 (https://jinja.palletsprojects.com/en/3.0.x/templates/#math)